index.js

```diff
@@ -298,16 +298,16 @@ function diffArguments(thisType, otherType, thisField, otherField, options) {
     let newArgsMap = getArgumentMap(otherField.args);
     oldArgsMap.forEach(function(value, key) {
         if (!newArgsMap.has(key)) {
-            const description = format('Argument `"{0} : {1}"` on type {2}.{3} has been removed from {4}.', value, key, thisType, thisField.name, options.labelForThis);
+            const description = format('Argument missing from {0} : `"{1} : {2}"` on field {3}.{4}.', options.labelForThis, key, value, thisType, thisField.name);
             diffs.push(new GraphQLDiff(thisType, otherType, DiffType.ArgDiff, description, false));
         } else if (newArgsMap.get(key) !== value) {
-            const description = format('Argument type changed from `"{0}: {1}"` : {2} to {3}: `"{4} : {5}"`', options.labelForThis, key, value, options.labelForOther, key, newArgsMap.get(key));
+            const description = format('Argument type diff on field {0}.{1}. {2}: `"{3} : {4}"` vs. {5} `"{6} : {7}."`', thisType, thisField.name, options.labelForThis, key, value, options.labelForOther, key, newArgsMap.get(key));
             diffs.push(new GraphQLDiff(thisType, otherType, DiffType.ArgDiff, description, false));
         }
     });
     newArgsMap.forEach(function(value, key) {
        if (!oldArgsMap.has(key)) {
-           const description = format('Argument `"{0} : {1}"` on type {2}.{3} has been added to {4}.', value, key, otherType, otherField.name, options.labelForOther);
+           const description = format('Argument missing from {0} : `"{1} : {2}"` on field {3}.{4}.', options.labelForOther, key, value, otherType, otherField.name);
```

Comment:
Use the same format as above: 

Argument missing from new schema: Query.VideoByGenre(genre: String!).